src/w2utils.js

```diff
@@ -247,7 +248,7 @@ var w2utils = (function ($) {
     }
 
     function isDateTime (val, format, retDate) {
-        if (format == null) format = w2utils.settings.dateFormat + '|' + w2utils.settings.timeFormat;
+        if (format == null) format = w2utils.settings.datetimeFormat;
```

Comment:
Users might want to have a different format for date, time and timetime. for example they could want to show seconds on time, but not on datetime.
Or the other way around, they might need a higher precision on datetime, but not on time.